src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,34 @@
+package racingcar.domain;
+
+import static racingcar.exception.ExceptionMessage.DUPLICATE_CARS;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars(List<String> cars) {
+        checkDuplicateCars(cars);
+        this.cars = cars.stream()
+                .map(Car::new)
+                .collect(Collectors.toList());
+    }
+
+    private void checkDuplicateCars(List<String> cars) {
+        Map<String, Long> duplicateCars = cars.stream()
+                .collect(Collectors.groupingBy(s -> s, Collectors.counting()))
+                .entrySet().stream()
+                .filter(entry -> entry.getValue() > 1)
+                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+
+        if(!duplicateCars.isEmpty()) {
+            throw new IllegalArgumentException(DUPLICATE_CARS.getMessage());
+        }
+    }
+
+    public List<Car> getCarList() {
+        return cars;

```

Comment:
여기도 불변객체로 반환하는게 일급 컬렉션 목적에 더 알맞을거같아요!

```suggestion
        return Collections.unmodifiableList(cars);
```